### PR TITLE
Update to 2020 boundaries, Mapping Inequality data, and move to {sf}

### PR DESCRIPTION
**Pull request by delosh653**
This is a significant package update, which does the following:

- Updates all boundaries to 2020
- updates Mapping Inequality data (several cities added)
- Switches from using {sp} and related geospatial packages, to {sf}, the updated standard for geospatial manipulation

### CODE REVIEW

File: README.md
Comment (karenmjiang) [inline README.md:1]: Do you want to add a short section on what updates are available in this new version in the readme? Basically port over what you have the the MR description.

File: inst/extdata/Specific_US_cities.csv
Comment (karenmjiang) [inline inst/extdata/Specific_US_cities.csv:1]: Is this the manual check file? Suggestion to add a small readme.txt in the `extdata` directory briefly describing the files that are in this directory. Very brief, maybe which ones are downloaded and a link, which ones are generated, etc.

File: README.md
Comment (delosh653) [reply README.md:1]: So I was going to put that in the release notes, which will explain changes, but I think 2020 is relevant to the user (the rest is somewhat under the hood changes), so I'll add that to the readme.

File: inst/extdata/Specific_US_cities.csv
Comment (delosh653) [reply inst/extdata/Specific_US_cities.csv:1]: done!

File: README.md
Comment (delosh653) [reply README.md:1]: done!

**Comment by karenmjiang** [comment]
Everything look good! Written approval for MR since there is no button :)

**Pull request merged**